app/views/retire_document/create.html.erb

```diff
@@ -0,0 +1,35 @@
+<% content_for :back_link, render_back_link(href: document_path(@document)) %>
+<% content_for :title, t("retire_document.retire.title", title: @document.title) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_tag(new_retirement_path(@document)) do %>
+      <%= render "govuk_publishing_components/components/textarea", {
```

Comment:
Not sure if it was clear from the card but we support (it probably wasn't) but we support govspeak as an input so we may need to ponder some sort of cutdown markdown editor for this (could totally be a future card but something to bear in mind)